spec/mailers/user_mailer_spec.rb

```diff
@@ -587,7 +587,6 @@ def expect_email_body_to_have_help_and_contact_links
     end
 
     let(:mail) do
-      p enrollment
```

Comment:
oooo it was `p` ... I was gonna say I think `puts` is caught by Rubocop

Reply:
PR to update our Rubocop config in https://github.com/18F/identity-idp/pull/7120